src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
Disclaimer -- not my code, so I'm guessing...

Took me a while to puzzle this out, but I think I now understand what they were going for here.  I think a better example would be to have something like:

```
{"myDateProperty": {"$date", <dateAsEpoch>}}
```

This both works with the code, is probably more what most folks will do, matches up with the "raw JSON" representation of dates in Mongo.  Basically the code is assuming you'll have the raw JSON object and it is looking for the `$date` property to give it special treatment.  However, since all it does is look for any field named `$date` it can lead to some odd constructions (such as the one in the example).

Following up on your first comment, I'm actually not sure this is what we'd want to do long term (or perhaps not even short term).  I think it would be more natural to serialize to/from Java Date instances (or perhaps since we're Java 8 here to the new `java.time` classes).  I might play around to see how difficult this would be.  At a minimum I agree that the example should be updated to use a call on one of these to get the milliseconds (which is what all the tests do).